README.md

```diff
@@ -10,7 +10,7 @@ Golint requires Go 1.6 or later.
 
 ## Usage
 
-Invoke `golint` with one or more filenames, a directory, or a package named
+Invoke `golint` with one or more filenames, directories, or packages named
```

Comment:
Hmm, how come this correctly says "directories", but commit 5295072 in master incorrectly says "**a** directories"?

@dsnet said in https://github.com/golang/lint/pull/271#issuecomment-277328459

> Merged as 5295072. I made some minor changes to your logic.

Was this an intentional change?